core/src/main/java/org/radargun/stats/Request.java

```diff
@@ -109,4 +111,12 @@ public long getResponseCompleteTime() {
    public long duration() {
       return responseCompleteTime - requestStartTime;
    }
+
+   public long getIntendedTime() {
+      return intendedTime;
+   }
+
+   public void setIntendedStartNs(long intendedTime) {
```

Comment:
let's match getter and setter name (if using `getFoo()`/`setFoo(long f)` style and not just `long foo()`/`void foo(long f)`)